Configurations/unix-Makefile.tmpl

```diff
@@ -844,6 +844,24 @@ tags TAGS: FORCE
 	-ctags -R .
 	-etags `find . -name '*.[ch]' -o -name '*.pm'`
 
+# Find the sources for the FIPS module ('providers/fips')
+# Only take the C and H files
+# Shave off $(SRCDIR) before processing
+fips-checksums: FORCE
+	if which unifdef > /dev/null; then \
```

Comment:
If you invert the test and "exit 1;" in the failure you can avoid indenting everything else.

Reply:
Good point.

Reply:
On the other hand, as long as we go with not punishing those who haven't installed `unifdef`, that won't make any difference.